_about/team.md

```diff
@@ -1,5 +1,4 @@
 ---
-permalink: /about/team.html
```

Comment:
Why remove the permalink?

Reply:
For the team page to be indexed like the others it has to have url /about/team/.